digiroad2-viite/src/test/scala/fi/liikennevirasto/viite/ProjectServiceSpec.scala

```diff
@@ -284,211 +287,6 @@ class ProjectServiceSpec  extends FunSuite with Matchers with BeforeAndAfter {
     } should have size (count - 1)
   }
 
-  test("Unchanged with termination test, repreats termination update, checks calibration points are cleared and moved to correct positions") {
```

Comment:
An awful lot of tests have been removed. On purpose?

Reply:
result of bad merge, thought those were deleted from master

Reply:
this test was deleted from Project *Link* ServiceSpec since we had it duplicated in ProjectServiceSpec.

Reply:
that is why those had familiar names. Not sure why those were marked as additions for me though.